libnmstate/nm/device.py

```diff
@@ -197,13 +197,6 @@ def _requires_activation(dev, connection_profile):
             dev.get_iface(),
         )
         return True
-    if _ipv6_changed(dev, connection_profile):
```

Comment:
After this change, nothing calls `_ipv6_changed()` anymore. @ffmancera please file a follow-up PR to remove its definition.